modules/model/shared/src/main/scala/navigate/model/enums/M1Source.scala

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package navigate.model.enums
+
+import lucuma.core.util.Enumerated
+
+/** Enumerated type for M1 Source. */
+sealed abstract class M1Source(val tag: String) extends Product with Serializable
+
+object M1Source {
+//  case object PWFS1 extends M1Source("PWFS1")
```

Comment:
Commented code

Reply:
YEs, but it will be included in the future. I commented it so I don't have to deal with that option until then.